scripts/sh-colo.sh

```diff
@@ -28,11 +28,43 @@ EOF
   log "add.roles ${_http_body}=|${_test}|"
 }
 
+function network_configure_shcolo() {
```

Comment:
I'm sure we can merge this in as a conditional stanza to the network_configure function.